classes/Product.php

```diff
@@ -2725,10 +2725,9 @@ public function getAttributesResume($id_lang, $attribute_value_separator = ' - '
             return false;
         }
 
-        $product_attributes = [];
-        foreach ($combinations as $combination) {
-            $product_attributes[] = (int) $combination['id_product_attribute'];
-        }
+        $combinations = array_column($combinations, null, 'id_product_attribute');
+
+        $product_attributes = array_keys($combinations);
```

Comment:
```suggestion
        $combinationIds = array_keys($combinations);
```
also don't forget to change bellow in the sql query.

Reply:
Thanks you. It's done.